examples/common.yaml

```diff
@@ -15,6 +15,8 @@ dlrn::workers:
     release: 'mitaka'
 #   gerrit_user: 'rdo-trunk'
 #   gerrit_email: 'rdo-trunk@rdoproject.org'
+    rsyncdest: '192.168.121.34:/home/centos-master/data/'
```

Comment:
I would prefer to have these new options commented out by default, so a first installation doesn't try to enable rsync.